api-reference/beta/api/device_post_registeredowners.md

```diff
@@ -16,56 +16,50 @@ One of the following permissions is required to call this API. To learn more, in
 ## HTTP request
```

Comment:
Per other feedback - it would be great to get more info from Jairo and Shantaram (or even Billy Jack) on the value of setting a registered owner.  What is the registered owner allowed to do, over a registeredUser?  What's the difference between these two, because it should be documented.